RogueEssence/Menu/Underfoot/ItemUnderfootMenu.cs

```diff
@@ -164,5 +164,45 @@ public override void Draw(SpriteBatch spriteBatch)
             if (summaryMenu != null)
                 summaryMenu.Draw(spriteBatch);
         }
+
+        protected override void UpdateKeys(InputManager input)
+        {
+            bool equippedItems = false;
+            foreach (Character character in DataManager.Instance.Save.ActiveTeam.Players)
+            {
+                if (!character.Dead && !String.IsNullOrEmpty(character.EquippedItem.ID))
+                {
+                    equippedItems = true;
+                    break;
+                }
+            }
+            bool invEnabled = !(DataManager.Instance.Save.ActiveTeam.GetInvCount() == 0 && !equippedItems);
+
+            if (IsInputting(input, Dir8.Left))
+            {
+                CheckSwitchMenu(true, invEnabled);
+            }
+            else if (IsInputting(input, Dir8.Right))
+            {
+                CheckSwitchMenu(false, invEnabled);
+            }
+            base.UpdateKeys(input);
+        }
+
+        private void CheckSwitchMenu(bool setEndPage, bool inventoryEnabled)
+        {
+            if (inventoryEnabled)
+            {
+                ItemMenu menu = new ItemMenu();
+                if (setEndPage)
+                {
+                    menu.SetPage(menu.TotalChoices.Length - 1);
```

Comment:
Instead of having to expose SetPage, just add a constructor override to ItemMenu that takes in a currently chosen slot. (this can future proof against cases where we want to open the inv menu with the cursor set on something.)  Make it override `defaultChoice` there.